images/php/01-bay.ini

```diff
@@ -20,3 +20,5 @@ disable_functions = ${BAY_DISABLE_FUNCTIONS}
 
 [xdebug]
 xdebug.mode = debug,profile
+xdebug.output_dir = /app/docroot/sites/default/files/xdebug-profile
```

Comment:
I'm slightly concerned about these ending up in the publicly available fs, could they be moved to private without affecting availability to other tools? I'm guessing not and that the output files are similar in content to a stack trace so not much of a concern but it's still a conversation I'd like to have.

Reply:
I think this will work if the files are in private. For local dev you  can see "private" files from the host which is the requirement. I'll test and update accordingly. Thanks for the review @GROwen! Even for non local, you'd probably copy them to a local system. The main requirement is just to get them out of the container.